src/main/target/REVO/target.c

```diff
@@ -32,10 +32,10 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM8,  CH3, PC8,  TIM_USE_PWM,   0, 0),  // S5_IN
     DEF_TIM(TIM8,  CH4, PC9,  TIM_USE_PWM,   0, 0),  // S6_IN
 #ifdef REVOLT
```

Comment:
The motor mapping is now identical to REVO, only LED on PB6 is unique for REVOLT. Condition can be simplified.